frontend/src/components/modals/settings/SettingsModal.tsx

```diff
@@ -1,202 +0,0 @@
-import { Spinner } from "@nextui-org/react";
```

Comment:
Is the other stuff in this directory still valid?

Reply:
Just removed two components that are no longer used and left one that is being used elsewhere. I do plan or re-organizing the directories better in the near-future.

There are probably some more services that may no longer be used but I'll limit this PR to the old session class and handle the rest in follow-ups